challenges/shift-array/shift_array.py

```diff
@@ -0,0 +1,29 @@
+def insertShiftArray(arr,val):
+    """
+    Parameters:
+        arr: a list
+        val: any data type
+    Returns:
+        an array that is arr with val inserted into the "center" index position
+    """
+    # Validate input
+    try:
+        if type(arr) != list: raise TypeError('First param not a list')
+    except TypeError as err:
+        print(err.args[0])
+        return False
+    
+    # Consider edge case: empty list
+    if not len(arr): return [val]
+
+    # Insert val at mean index or mean index + .5
```

Comment:
I like that you add these comments in telling me as a programmer what this code is trying to do. Nice!